status-im/status-react
# [fix #4819] update react-native to 0.56

fixes #4819 

## Summary

update status-react mobile version to react-native 0.56

## Step for testing

verify that the two bugs introduced in previous update are gone:
- perf regression related to chat input element (#4794)
- keyboard crash issue (#4818)
- make sure that Korean input on iOS works correctly

status: ready
